components/meter/meter-radial.js

```diff
@@ -0,0 +1,52 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { heading4Styles } from '../typography/styles.js';
+
+class MeterRadial extends LitElement {
+	static get styles() {
+		return [ heading4Styles, css`
+		:host {
+			display: block;
+		}
+		.d2l-meter-radial-full-bar,
+		.d2l-meter-radial-progress-bar {
+			fill: none;
+			stroke-linecap: round;
+			stroke-width: 9;
+		}
+		.d2l-meter-radial-full-bar {
+			stroke: var(--d2l-color-gypsum);
+		}
+		.d2l-meter-radial-progress-bar {
+			stroke: var(--d2l-color-celestine);
+		}
+		.d2l-meter-radial-text {
+			color: var(--d2l-color-ferrite);
+		}
+	` ];
+	}
+
+	render() {
+		const lengthOfLine = 115; // found by approximating the half the perimeter of the ellipse with radii 38 and 35
+		const progressFill = this.value / this.max * lengthOfLine;
+		return html `
+			<svg viewBox="-5 -47 86 52" width="86" height="52">
```

Comment:
I personally find non-0 origin viewboxes confusing... I found a neat trick to get it to 0 without having to do crazy maths if you're interested.

Reply:
Yeah, I used the viewBox to define where the origin is to make the rest easier. If there is a better way to do it I would love to know :). (Although, doing crazy maths in fun -- my background is Pure Math)

Reply:
In the past I've used [SVG-OMG](https://jakearchibald.github.io/svgomg/) and then modified everything inside the SVG to have a `translate` of the inverse of the viewBox. SVGOMG then optimizes it.

So like:
```
<svg viewBox="0 0 86 52" ...>
  <g transform="translate(5 47)">
      <path>...</path>
  </g>
</svg>
```

Not sure if that'll work for your with the text inside or not... and it may not be any easier to understand!

Reply:
Interesting. I'll do that.